agent_api_rest/src/verification/authorization_requests.rs

```diff
@@ -24,32 +25,58 @@ pub(crate) async fn get_authorization_requests(
     // Get the authorization request if it exists.
     match query_handler(&authorization_request_id, &state.query.authorization_request).await {
         Ok(Some(AuthorizationRequestView {
-            siopv2_authorization_request: Some(siopv2_authorization_request),
+            authorization_request: Some(authorization_request),
             ..
-        })) => (StatusCode::OK, Json(siopv2_authorization_request)).into_response(),
+        })) => (StatusCode::OK, Json(authorization_request)).into_response(),
         Ok(None) => StatusCode::NOT_FOUND.into_response(),
         _ => StatusCode::INTERNAL_SERVER_ERROR.into_response(),
     }
 }
 
+#[derive(Deserialize, Serialize)]
+pub struct AuthorizationRequestsRequest {
```

Comment:
Is this naming intentional? 😄

Reply:
It is intentional, but admittedly quite confusing :sweat_smile: It's just a tool for deserializing, just like `CredentialsRequest` for the `/v1/credentials` endpoint. 

I will change it to `AuthorizationRequestsEndpointRequest` to make it less confusing :+1: